docs/Developer Resources/Custom Packages.rst

```diff
@@ -207,7 +206,6 @@ packages can be built as follows:
 
 .. code:: sh
 
-   $ sudo apt-get install dkms
```

Comment:
Looks like it's only needed for native debs, I propose to add it only here.

In CI we build non-native packages successfully without dh-dkms https://github.com/openzfs/zfs/blob/master/.github/workflows/build-dependencies.txt

Reply:
Will change. I added `gdebi-core` because the command to install uses it:

```
for file in *.deb; do sudo gdebi -q --non-interactive $file; done
```

Reply:
I also think this might be a better way to install:

```
find . -maxdepth 1 -name '*.deb' -print0 | sudo xargs -0 --verbose apt-get install
```

Reply:
I think we don't need gdebi at all and can just use apt-get or dpkg. If you're ready to do it - I propose to remove gdebi at all on this page and use `apt-get`.

find + xargs duo is a little bit less readable, I'd prefer to leave plain bash loop here.

Reply:
I tried to do `apt-get install *.deb`, but it seems that just the file names without a path prefix (`./` in this case) doesn't work. I'll try it again tomorrow, maybe I did something wrong.

Reply:
https://github.com/openzfs/zfs/blob/master/.github/workflows/scripts/setup-dependencies.sh#L42 How we use it in our CI

Reply:
Oh, nice :) Somehow I came up with this long line instead of this simple way :)

Is this running on Ubuntu? Because somehow I had an issue with the native build conflicting with `zfs-dkms` which the Ubuntu kernel provides, but after removing the `Conflicts:` line from the package everything works.

Reply:
Updated

Reply:
> Is this running on Ubuntu?  

yes, 18.04 and 20.04 iirc

Reply:
Maybe I need to raise an issue in the ZFS repo then, when trying to install `native-deb-utils`'s `openzfs-zfsutils` on Ubuntu 23.04 I always got conflicts because the package has `Conflicts: zfs-dkms` and Ubunut's kernel images provide `zfs-dkms`. I removed the line from the `contrib/debian/control` and the package installed and works just fine.

Reply:
I thought that I already saw such issue, but didn't find it, feel free to use https://github.com/openzfs/zfs/issues for that

Reply:
I searched already but wasn't able to find anything, I'll create an issue for that tomorrow.

Reply:
@EchterAgo found it https://github.com/openzfs/zfs/issues/15404 (at least in comments)